src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import core.basesyntax.handler.BalanceOperationHandler;
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.handler.PurchaseHandler;
+import core.basesyntax.handler.ReturnHandler;
+import core.basesyntax.handler.SupplyHandler;
+import core.basesyntax.implementation.FruitShopServiceImpl;
+import core.basesyntax.implementation.ParserServiceImpl;
+import core.basesyntax.implementation.ReaderServiceImpl;
+import core.basesyntax.implementation.ReportGeneratorServiceImpl;
+import core.basesyntax.implementation.WriterServiceImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.ParserService;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportGeneratorService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    private static final String INPUT_FILE =
+            "src/main/java/resources/database";
+    private static final String REPORT_FILE =
+            "src/main/java/resources/ReportFile";
```

Comment:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/10d39b72-71ea-4350-96a8-72449d5b154a)
src/main/**java**/ - for java code!